### DISCUSSION TITLE
What does the g command exactly do?

### DISCUSSION THREAD

**Discussion by mgpinf** [nushell/nushell#4270] [Q&A]
What does the g command exactly do? In documentation it says that we can jump between shells but I'm unable to understand how exactly the command would work

**Comment by kubouch** [comment]
That's for switching between shells. "Shells" is a functionality that lets you have multiple sessions open at the same time.

You can enter a new shell with `enter`, for example:
```
> enter ..
```

Then, `shells` will show you two directories, numbered 0 and 1.
 
You can switch between the directories with `n`, `p`, or `g <number>` where `<number>` is the number you see in `shells` command output.
